libs/application/core/src/types/ApplicationTypes.ts

```diff
@@ -35,6 +36,10 @@ export const ApplicationConfigurations = {
     slug: 'okuskirteini',
     translation: 'dl.application',
   },
+  [ApplicationTypes.DRIVING_ASSESSMENT_APPROVAL]: {
+    slug: 'akstursmat',
+    translation: 'dl.application',
```

Comment:
`DRIVING_LESSONS`, `DRIVING_LICENSE ` and `DRIVING_ASSESSMENT_APPROVAL` all belong to the same namespace `dl`, is this intentional since this might cause messages to conflict in Contentful if any messages fields have the same keys.  
I'm not sure how conflicts are handled inside the export strings function, this might fail safe.

Reply:
I think a single "extract-strings" command needs to find all strings belonging to a namespace, so depending on how they're, it might be more appropriate to have separate namespaces for each template.

Reply:
Regarding messages with the same key, if the message has already been synced once with Contentful, it won't override the Icelandic and English translations. However it will change the defaultMessage and Description. Also, these two objects might expect different messages for the 2 different templates, but only one of them will live, which is the one with the latest changes on Contentful.

I don't know how much each of these templates are sharing messages, but it's easier to have 1 namespace by the template to avoid conflict and some surprise when checking the app and finding the wrong copy from another template